pages.fr/linux/ascii.md

```diff
@@ -0,0 +1,36 @@
+# ascii
+
+> Affiche les alias de caractères ASCII.
+> Plus d'information : <https://www.catb.org/~esr/ascii/>.
+
+- Affiche les alias ASCII d'un caractère :
+
+`ascii {{c}}`
+
+- Affiche les alias ASCII dans un format court, adapté pour les scripts :
```

Comment:
Shouldn't this `adapté pour les scripts` be `script-friendly mode`'s French equivalent?

Note: I checked this with Google Translate

Reply:
Uh yes it is, but I don't understand the underlying question.

Reply:
Now, I checked these words alone in Google translate and it read `suitable for scripting` so I guess it's fine. I will mark this conversation as resolved.

I used the translate to English option for the whole page in chrome, I guess there must have been an issue there.